accord-core/src/main/java/accord/primitives/TxnId.java

```diff
@@ -120,7 +128,7 @@ private static Domain domain(int flags)
 
     private static int rwOrdinal(int flags)
     {
-        return (flags >> 1) & 1;
+        return (flags >> 1) & 7;
```

Comment:
not specific to this patch, but could we rename `rw` -> `kind`?

Reply:
We have a bunch of `Kind`, but I agree `rw` is now bad terminology. I'll ponder what alternatives there might be, and same for the name within Txn